extensions/wsc/src/main/java/org/mule/extension/ws/internal/xml/util/XMLUtils.java

```diff
@@ -28,17 +29,20 @@
 import javax.xml.transform.Source;
 import javax.xml.transform.Transformer;
 import javax.xml.transform.TransformerConfigurationException;
+import javax.xml.transform.TransformerException;
 import javax.xml.transform.TransformerFactory;
 import javax.xml.transform.dom.DOMResult;
 import javax.xml.transform.dom.DOMSource;
 import javax.xml.transform.sax.SAXSource;
+import javax.xml.transform.stream.StreamResult;
 import javax.xml.transform.stream.StreamSource;
 
+import net.sf.saxon.jaxp.SaxonTransformerFactory;
```

Comment:
is this correct? Does the extension have a direct dependency to Saxon on its pom or is it trusting the runtime to export it?